x-pack/plugins/security_solution/server/endpoint/routes/resolver/queries/events.ts

```diff
@@ -78,20 +131,34 @@ export class EventsQuery {
   }
 
   /**
-   * Searches ES for the specified events and format the response.
+   * Will search ES using a filter for normal events associated with a process, an e
    *
    * @param client a client for searching ES
    * @param filter an optional string representation of a raw Elasticsearch clause for filtering the results
    */
   async search(
     client: IScopedClusterClient,
-    filter: string | undefined
+    body: { filter?: string; eventID?: string; entityType?: string },
+    alertsClient: AlertsClient
   ): Promise<SafeResolverEvent[]> {
-    const parsedFilters = EventsQuery.buildFilters(filter);
-    const response = await client.asCurrentUser.search<SafeResolverEvent>(
-      this.buildSearch(parsedFilters)
-    );
-    // @ts-expect-error @elastic/elasticsearch _source is optional
-    return response.hits.hits.map((hit) => hit._source);
+    if (body.filter) {
+      const parsedFilters = EventsQuery.buildFilters(body.filter);
+      const response = await client.asCurrentUser.search<SafeResolverEvent>(
+        this.buildSearch(parsedFilters)
+      );
+      // @ts-expect-error @elastic/elasticsearch _source is optional
+      return response.hits.hits.map((hit) => hit._source);
```

Comment:
Note to self, create an issue for migrating analyzer's use of `_source` to `fields`